### ISSUE TITLE
Where can I download the annotations?

### ISSUE THREAD

**Issue by elmtang** [bircatmcri/MCIndoor20000#1]
Hello,

I'm interested in using MCIndoor2000 data for my project, I wonder where can I download the annotations of images?

Thanks,
Howard

**Comment by elmtang** [comment]
Got it, no annotation available in this dataset.

**Comment by aptafti** [comment]
Yes, it is just very general annotations, like door, stairs ... not
detailed annotations.
